From 396361d2cfc200cd95e67a1d8b4b4db70c0ec147 Mon Sep 17 00:00:00 2001
From: Tony Camuso <tcamuso@redhat.com>
Date: Fri, 8 Aug 2014 18:14:37 -0400
Subject: [char] ipmi: Initialize locals to avoid warning

Message-id: <1407521687-4858-4-git-send-email-tcamuso@redhat.com>
Patchwork-id: 87544
O-Subject: [RHEL7.1 PATCH 03/13] ipmi: Initialize locals to avoid warning
Bugzilla: 1111214
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1111214

Cherry-picked without conflicts from the following upstream commit.

commit f0ba93907b5a7c3c87b6baae1260a254ecfd6f98
Author: Corey Minyard <cminyard@mvista.com>
Date:   Thu Sep 5 06:36:34 2013 -0500

    ipmi: Initialize locals to avoid warning

    A couple of variables were getting warnings about being uninitialized.
    It was a false warning, but initialize them, anyway.

    Signed-off-by: Corey Minyard <cminyard@mvista.com>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

Signed-off-by: Tony Camuso <tcamuso@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/char/ipmi/ipmi_msghandler.c b/drivers/char/ipmi/ipmi_msghandler.c
index 4445fa1..ec4e10f 100644
--- a/drivers/char/ipmi/ipmi_msghandler.c
+++ b/drivers/char/ipmi/ipmi_msghandler.c
@@ -1848,7 +1848,7 @@ int ipmi_request_settime(ipmi_user_t      user,
     int              retries,
     unsigned int     retry_time_ms)
 {
- unsigned char saddr, lun;
+ unsigned char saddr = 0, lun = 0;
  int           rv;
 
  if (!user)
-- 
1.7.1